test/dubbo/dubbo2.6/pom.xml

```diff
@@ -0,0 +1,38 @@
+<!--
```

Comment:
The path `test/dubbo/dubbo2.6` is incorrect. It should be `test/dubbo/dubbo-2.6`.